### PR TITLE
[terra-avatar] Update docs to use MDX syntax

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
Updated docs to use MDX syntax.

### CODE REVIEW

File: packages/terra-avatar/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ChangeLog
 =========
+### Changed
```
Comment (emilyrohrbough) [inline packages/terra-avatar/CHANGELOG.md]: oops...this needs moved under the unreleased section

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: whoops, quite a few extra empty lines here

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/UpgradeGuide.5.doc.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+<Badge />
+
+# terra-avatar Upgrade Guide
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/UpgradeGuide.5.doc.mdx]: It looks like we copied over the content from docs/upgrade-guide.md/ Do we need to delete `'../../../../docs/upgrade-guide.md';`?

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import SharedUser from '../example/shared-user/SharedUser?dev-site-example';
+import SharedUserSize from '../example/shared-user/SharedUserSize?dev-site-example';
+import SharedUserColorVariants from '../example/shared-user/SharedUserColorVariants?dev-site-example';
+
+import SharedUserPropsTable from '../../../../src/variants/SharedUser?dev-site-props-table';
+
+<Badge />
+
+# Shared User
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx]: It looks like we copied over the content from `docs/shared-user.md/.`Do we need to delete `'../../../../docs/shared-user.md';`?

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import Facility from '../example/facility/Facility?dev-site-example';
+import FacilityImage from '../example/facility/FacilityImage?dev-site-example';
+import FacilitySize from '../example/facility/FacilitySize?dev-site-example';
+import FacilityColorVariants from '../example/facility/FacilityColorVariants?dev-site-example';
+
+import FacilityPropsTable from '../../../../src/variants/Facility?dev-site-props-table';
+
+# Facility
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx]: It looks like we copied over the content from `docs/f.md/.`Do we need to delete `'../../../../docs/facility.md';`?

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+import AvatarImage from '../example/avatar/AvatarImage?dev-site-example';
+import AvatarIsDeceased from '../example/avatar/AvatarIsDeceased?dev-site-example';
+import AvatarSize from '../example/avatar/AvatarSize?dev-site-example';
+import AvatarColorVariants from '../example/avatar/AvatarColorVariants?dev-site-example';
+
+import AvatarPropsTable from '../../../../src/variants/Avatar?dev-site-props-table';
+
+<Badge />
+
+# Terra Avatar
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx]: It looks like we copied over the content from `docs/avatar.md/`. Do we need to delete `'../../../../docs/avatar.md';`?

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+<Badge />
+
+# About terra-avatar
```
Comment (emilyrohrbough) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: It looks like we copied over the content from `docs/README.md/`. Do we need to delete `'../../../../docs/README.md';`?

Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: Oops I didn't see it. I will change it right now.

File: packages/terra-avatar/CHANGELOG.md (hunk 1 above)
Comment (pranav300) [reply packages/terra-avatar/CHANGELOG.md]: updated here https://github.com/cerner/terra-core/pull/2810/commits/ade62bfd9f0af64bf09835d5432378e52191c617.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx (hunk 2 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/UpgradeGuide.5.doc.mdx (hunk 3 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/UpgradeGuide.5.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx (hunk 4 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx (hunk 5 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx (hunk 6 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx (hunk 7 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx [hunk 8]
```diff
@@ -1,3 +1,7 @@
+import { Badge } from '../../../../package.json?dev-site-package';
```
Comment (mjhenkes) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]:
```suggestion
import { Badge } from 'terra-avatar/package.json?dev-site-package';
```

Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: https://github.com/cerner/terra-core/pull/2810/commits/f949c1d7652e7604a5dbf1e3bc3675e3f8aa2200.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx [hunk 9]
```diff
@@ -17,32 +21,17 @@ This component requires the following peer dependencies be installed in your app
...
-
-<SharedUser
-  alt="Shared-User"
-/>
 ```
 
 ## Component Features
 
  * [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
  * [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
```
Comment (supreethmr) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: this links seems to broken across all the components. is there any issue logged to update these links..? if not you can update these links here.

Comment (nramamurth) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: Here's a conversation about it - https://github.com/cerner/terra-ui/issues/172#issuecomment-548389333. I think this is the right time for us to use deployed site links in place of GitHub links.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+import AvatarUser from '../example/avatar/AvatarUser?dev-site-example';
+import AvatarOneInitial from '../example/avatar/AvatarOneInitial?dev-site-example';
+import AvatarTwoInitials from '../example/avatar/AvatarTwoInitials?dev-site-example';
+import AvatarImage from '../example/avatar/AvatarImage?dev-site-example';
+import AvatarIsDeceased from '../example/avatar/AvatarIsDeceased?dev-site-example';
+import AvatarSize from '../example/avatar/AvatarSize?dev-site-example';
+import AvatarColorVariants from '../example/avatar/AvatarColorVariants?dev-site-example';
+
+import AvatarPropsTable from '../../../../src/variants/Avatar?dev-site-props-table';
```
Comment (mjhenkes) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx]:
```suggestion
import AvatarPropsTable from 'terra-avatar/src/variants/Avatar?dev-site-props-table';
```

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/ChangeLog.6.doc.mdx [hunk 11]
```diff
@@ -0,0 +1,7 @@
+import { Badge } from 'terra-avatar/package.json?dev-site-package';
+
+import ChangeLog from '../../../../CHANGELOG.md';
```
Comment (mjhenkes) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/ChangeLog.6.doc.mdx]:
```suggestion
import ChangeLog from 'terra-avatar/CHANGELOG.md';
```

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx [hunk 12]
```diff
@@ -0,0 +1,46 @@
+import { Badge } from 'terra-avatar/package.json?dev-site-package';
+
+import Facility from '../example/facility/Facility?dev-site-example';
+import FacilityImage from '../example/facility/FacilityImage?dev-site-example';
+import FacilitySize from '../example/facility/FacilitySize?dev-site-example';
+import FacilityColorVariants from '../example/facility/FacilityColorVariants?dev-site-example';
+
+import FacilityPropsTable from '../../../../src/variants/Facility?dev-site-props-table';
```
Comment (mjhenkes) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx]:
```suggestion
import FacilityPropsTable from 'terra-avatar/src/variants/Facility?dev-site-props-table';
```

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx [hunk 13]
```diff
@@ -0,0 +1,44 @@
+import { Badge } from 'terra-avatar/package.json?dev-site-package';
+
+import SharedUser from '../example/shared-user/SharedUser?dev-site-example';
+import SharedUserSize from '../example/shared-user/SharedUserSize?dev-site-example';
+import SharedUserColorVariants from '../example/shared-user/SharedUserColorVariants?dev-site-example';
+
+import SharedUserPropsTable from '../../../../src/variants/SharedUser?dev-site-props-table';
```
Comment (mjhenkes) [inline packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx]:
```suggestion
import SharedUserPropsTable from 'terra-avatar/src/variants/SharedUser?dev-site-props-table';
```

Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx (hunk 12 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/ChangeLog.6.doc.mdx (hunk 11 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/ChangeLog.6.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx (hunk 10 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/Avatar.2.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.

File: packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx (hunk 9 above)
Comment (pranav300) [reply packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2810/commits/801b0a84866f2abeb6c5187ee1b8a042b9e3d76b.

**Pull request merged**